config/config_test.go

```diff
@@ -93,7 +93,8 @@ func TestGetExternalConfig(t *testing.T) {
 	noEntConfig := []byte(`
 {
 "customer_key": "",
-"gen_provider": "onprem"	
+"gen_platform": "vagrant-virtualbox",
```

Comment:
What if the user is using Fusion or another vbox provider? Do we really want to hard code this?